packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -2462,3 +2464,351 @@ Specific property types support a subset of [JSON Schema validations](https://to
+                    "title": "Primary email",
+                    "type": "string",
+                    "required": true,
+                    "format": "email",
+                    "permissions": [
+                        {
+                            "principal": "SELF",
+                            "action": "READ_WRITE"
+                        }
+                    ]
+                }
+            },
+            "required": [
+                "login",
+                "firstName",
+                "lastName",
+                "email"
+            ]
+        },
+        "custom": {
+            "id": "#custom",
+            "type": "object",
+            "properties": {
+              "twitterUserName": {
+                  "title": "Twitter username",
+                  "description": "User's username for twitter.com",
+                  "type": "string",
+                  "required": false,
+                  "unique": "UNIQUE_VALIDATED",
+                  "minLength": 1,
+                  "maxLength": 20,
+                  "permissions": [
+                      {
+                          "principal": "SELF",
+                          "action": "READ_WRITE"
+                      }
+                  ]
+              }
+            },
+            "required": []
+        }
+    },
+    "type": "object",
+    "properties": {
+        "profile": {
+            "allOf": [
+                {
+                    "$ref": "#/definitions/base"
+                },
+                {
+                    "$ref": "#/definitions/custom"
+                }
+            ]
+        }
+    }
+}
+```
+
+### Update existing custom property to be unique
+
+You can use the [update user profile schema property](#update-user-profile-schema-property) request or the [update group profile schema property](#update-group-profile-schema-property) request to mark existing custom user or group properties as unique, by specifying `"unique": true` on the properties that need to be marked as unique.
+
+Once the request to mark existing custom properties as unique is submitted, an asynchronous validation check is performed to make sure that there are no existing duplicate entries. If you have significant user or group records, the validation can take some time.
+
+A uniqueness status `"unique": "PENDING_UNIQUENESS"` indicates that the validation check is still in progress. You can go to the Universal Directory UI page to track the status of the validation check. Once the validation completes, if you submit a [get user schema](#get-user-schema) request or a [get group schema](#get-group-schema) request the property's uniqueness status will be changed to `UNIQUE_VALIDATED` if no duplicate records are found and the property will be uniqueness enforced. Otherwise if duplicate records are found, the `unique` attribute of the schema property will not be shown in the get schema request and uniqueness will not be enforced on the schema property.
```

Comment:
```suggestion
A uniqueness status of `"unique": "PENDING_UNIQUENESS"` indicates that the validation check is still in progress. Use the Universal Directory page in the Admin Console (**Directory** > **Directory Integrations**) to track the status of the validation check. After the validation completes, if you submit a [get user schema](#get-user-schema) request or a [get group schema](#get-group-schema) request, the property's uniqueness status changes to `UNIQUE_VALIDATED` if no duplicate records are found. The property is then uniqueness enforced. Otherwise, if duplicate records are found, the `unique` attribute of the schema property isn't shown in the get schema request and uniqueness isn't enforced on the schema property.
```

Reply:
Building on Susan's version: "The property is then uniqueness enforced." --> ", and uniqueness is then enforced on that property."